nix/haskell.nix

```diff
@@ -92,6 +92,14 @@ let
 
           # cardano-node socket path becomes too long otherwise
           unit.preCheck = lib.optionalString stdenv.isDarwin "export TMPDIR=/tmp";
+
+          # Force more integration tests to run in parallel than the
+          # default number of build cores.
+          #
+          # TODO: Figure out why this doesn't work in hydra when it works in buildkite,
+          # and enable.
```

Comment:
The `-j3` flag _should_ be added by this option when running `integration`.
Perhaps there are other things going on. Nix has two settings controlling parallelism - num cores for a build and num parallel builds - perhaps these are overriding our setting.
We can sort it later.